iothub_client/tests/common_longhaul/iothub_client_common_longhaul.c

```diff
@@ -936,7 +936,7 @@ static int on_message_received(void* context, const char* data, size_t size)
         }
         else
         {
-            LogError("on_message_received() failed (tests_id:%s, iotHubLonghaul->test_id:%s)", tests_id, iotHubLonghaul->test_id);
+            LogError("on_message_received() failed (deviceId:%s)", iotHubLonghaul->deviceInfo->deviceId);
```

Comment:
Why not keeping the test id and adding device id? too verbose? Better keep it to help correlate.

Reply:
I felt it was too verbose. It did not help me debug anything so I left only the relevant info.